streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/master/scheduler/controller/WorkerLifecycleController.java

```diff
@@ -0,0 +1,188 @@
+        .setMaxReconstructions(ActorCreationOptions.INFINITE_RECONSTRUCTION)
+        .createActorCreationOptions();
+
+    RayActor<JobWorker> actor = null;
+    if (Language.JAVA == language) {
+      // TODO (datayjz)
+    } else {
+      // TODO (datayjz)
+    }
+    
+    if (null == actor) {
+      LOG.error("Create worker actor failed.");
+      return false;
+    }
+
+    executionVertex.setWorkerActor(actor);
+
+    LOG.info("Create worker actor succeeded, actor: {}, vertex: {}.",
+        executionVertex.getWorkerActorId(), executionVertex.getVertexName());
+    return true;
+  }
+
+  /**
+   * Using context to init JobWorker.
+   *
+   * @param vertexToContextMap target JobWorker actor
+   * @param timeout timeout for waiting, unit: ms
+   * @return initiation result
+   */
+  public boolean initWorkers(
+      Map<ExecutionVertex, JobWorkerContext> vertexToContextMap, int timeout) {
+    LOG.info("Start to init workers: {}.", vertexToContextMap);
+    long startTime = System.currentTimeMillis();
+
+    Map<RayObject<Boolean>, ActorId> rayObjects = new HashMap<>();
+    vertexToContextMap.entrySet().forEach((entry -> {
+      ExecutionVertex vertex = entry.getKey();
+      rayObjects.put(RemoteCallWorker.initWorker(vertex.getWorkerActor(), entry.getValue()),
+          vertex.getWorkerActorId());
+    }));
+
+    List<RayObject<Boolean>> rayObjectList = new ArrayList<>(rayObjects.keySet());
+
+    LOG.info("Start to wait for workers' initialization.");
+    WaitResult<Boolean> result = Ray.wait(rayObjectList, rayObjectList.size(), timeout);
+    if (result.getReady().size() != rayObjectList.size()) {
+      LOG.error("Initialize workers timeout[{} ms].", timeout);
+      return false;
+    }
+
+    LOG.info("Finish waiting workers' initialization.");
+    LOG.info("Finish initializing workers. Cost {} ms.", System.currentTimeMillis() - startTime);
+    return true;
+  }
+
+  /**
+   * Start JobWorkers to run task.
+   *
+   * @param executionGraph physical plan
+   * @param timeout timeout for waiting, unit: ms
+   * @return starting result
+   */
+  public boolean startWorkers(ExecutionGraph executionGraph, int timeout) {
+    LOG.info("Start to start workers.");
+    long startTime = System.currentTimeMillis();
+    List<RayObject<Boolean>> rayObjects = new ArrayList<>();
+
+    // start source actors 1st
+    executionGraph.getSourceActors()
+        .forEach(actor -> rayObjects.add(RemoteCallWorker.startWorker(actor)));
+
+    // then start non-source actors
+    executionGraph.getNonSourceActors()
+        .forEach(actor -> rayObjects.add(RemoteCallWorker.startWorker(actor)));
+
+    WaitResult<Boolean> result = Ray.wait(rayObjects, rayObjects.size(), timeout);
+    if (result.getReady().size() != rayObjects.size()) {
+      LOG.error("Start workers timeout[{} ms].", timeout);
+      return false;
+    }
+
+    LOG.info("Start workers succeeded. Cost {} ms.", System.currentTimeMillis() - startTime);
```

Comment:
```suggestion
    LOG.info("Workers started. Costed {} ms.", System.currentTimeMillis() - startTime);
```

Reply:
Same modification on create, init and destroy.